File: kaninavl/kaninavl.cpp
#include <iostream>
using namespace std;

fib(int n) {
    if (n<=1) {
        return n;
    }
    return fib(n-1) + fib(n-2);
}

int main() {
    int n;
    cin >> n;

    for (int i=1; i<n+1; i++) {
        cout << fib(i) << endl;
    }
    return 0;
}

File: kaninavl/kaninavl.py
n = int(input())


def fib(n):
    fibs = [0, 1]
    for _ in range(2, n+1):
        fibs.append(fibs[-1] + fibs[-2])
    return fibs[1:]


for el in fib(n):
    print(el)
